src/modules/components/dao/daoDataListItemStructure/daoDataListItemStructure.tsx

```diff
@@ -0,0 +1,48 @@
+import type React from 'react';
+import { DataList, Heading, Icon, IconType, type IDataListItemProps } from '../../../../core';
+import { DaoAvatar } from '../daoAvatar';
+
+export interface IDaoDataListItemStructureProps extends IDataListItemProps {
+    name?: string;
+    logoSrc?: string;
+    description?: string;
+    address?: string;
+    ens?: string;
+    plugin?: string;
+    network?: string;
+}
+
+export const DaoDataListItemStructure: React.FC<IDaoDataListItemStructureProps> = (props) => {
+    const { name, logoSrc, description, network, plugin = 'token-based', address, ens, ...otherProps } = props;
+
+    return (
+        <DataList.Item {...otherProps}>
+            <div className="space-y-4">
+                <div className=" flex w-full justify-between">
+                    <div className="space-y-1.5 text-neutral-800">
+                        <Heading size="h2" as="h1">
+                            {name}
+                        </Heading>
+                        <Heading size="h4" as="h2">
+                            {ens ?? address}
+                        </Heading>
+                    </div>
+                    <DaoAvatar {...{ name, src: logoSrc }} size="md" />
```

Comment:
This is clearer instead of destructuring an object
```suggestion
                    <DaoAvatar name={name} src={logoSrc} size="md" />
```